common-pipeline/parameters/parameters-v3-schema.json

```diff
@@ -33,95 +35,42 @@
                "description": "The description of the parameter.",
                "type": "string"
             },
-            "paths": {
-               "description": "The locations of the parameterized properties in the pipeline.",
-               "items": {
-                  "type": "string"
-               },
-               "type": "array"
-            },
-            "type": {
+            "type_code": {
                "description": "The parameter type.",
                "enum": [
                   "string",
-                  "integer",
-                  "object",
-                  "array",
-                  "boolean"
+                  "encrypted",
```

Comment:
Where did these types come from?  These don't match the types of other data types in the pipeline flow schemas.  If this needs to be open then we shouldn't use an enum.

Reply:
@matthoward366 
Matt, 
Here is the list of parameter type for both local parameter and parameter in parameter set.

![image](https://user-images.githubusercontent.com/64796288/109522945-a8f65900-7a63-11eb-98fd-cad6cfa2ac9b.png)

Why the parameter type has to match the ones used in pipeline flow ?

Reply:
Why does this have to be an enum even?  Remember this is for anyone to use not just your product.

Reply:
On top of that why not keep this as `type`.  That match `subtype` below better.

Reply:
@matthoward366 
Matt,
Changes are made. Please review again. Thanks